graphics/vulkan-tools/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        KhronosGroup Vulkan-Tools 1.1.112 v
+name                vulkan-tools
+
+categories          graphics
+platforms           darwin
+license             Apache-2
+
+maintainers         {itanimul.li:josh @eintw1ck}
+description         Vulkan tools
+long_description    Provides vulkaninfo command for checking Vulkan installation configuration status
+
+depends_build-append port:vulkan-headers port:python37
```

Comment:
[minor] This violates the mode line. Please adjust the spacing. Commonly in this case we add a slash at the end of the line and move the ports to the next line. Also, please only list one port per line and also sort them by name. So change this to the following:

```
depends_build-append        \
                            port:python37 \
                            port:vulkan-headers
```